app/main.py

```diff
@@ -1,25 +1,84 @@
 from abc import ABC
+from typing import Type, Any
```

Comment:
sort imports

Reply:
<img width="335" alt="Screenshot 2023-05-24 at 20 14 17" src="https://github.com/mate-academy/py-aquapark/assets/114707574/ccb3af5d-b87e-4ca1-8b00-55eac3a2d93e">
Thanks for the feedback. Do you mean that I should include a blank line between the imports, similar to what is shown in the screenshot?

Reply:
I re-checked order of imports lesson and can't figure out what is wrong with these imports:
<img width="326" alt="Screenshot 2023-05-24 at 20 20 52" src="https://github.com/mate-academy/py-aquapark/assets/114707574/b2c09167-9304-4289-9ead-2445006d01e4">